modin/pandas/test/test_groupby.py

```diff
@@ -124,13 +124,28 @@ def test_mixed_dtypes_groupby(as_index):
             pandas_groupby = pandas_df.set_index(by[0]).groupby(
                 by=by[-1], as_index=as_index
             )
+            # difference in behaviour between .groupby().ffill() and
+            # .groupby.fillna(method='ffill') on duplicated indices
+            # caused by https://github.com/pandas-dev/pandas/issues/43412
+            # is hurting the tests, for now sort the frames
```

Comment:
may we raise a warning about this in the related methods? A user may be aware of this problem in pandas and so purposely use `.fiill/.bfill` not realizing that the problem persists for these methods as well in modin

Reply:
Good point, warning added.